src/App.jsx

```diff
@@ -87,6 +87,7 @@ function App() {
         const studyId = query.get('studyID');
         if (participantId) setParticipantID(participantId);
         if (studyId) setStudyID(studyId);
+        // console.log('using firebase', participantId, studyId);
```

Comment:
Do we still need these console log comments? If you think they'll be useful in the future, let's keep them. Otherwise, maybe we can cut them?

Reply:
Hey Ryan, yea I deleted most of them but decided to keep some that I thought would be helpful for debugging in the future. For the sake of simplification, we simply reversed the call of participant and study ID in some places, but ideally we should go back through and change the ordering throughout all of the files at some point. Let me know if you have any issues with keeping these in though.

Reply:
That's fair. So is this PR ready to go?